docs/faq.md

```diff
@@ -280,3 +280,27 @@ Note: Windows with Radeon GPUs currently default to 1 model maximum due to limit
 ## How does Ollama load models on multiple GPUs?
 
 Installing multiple GPUs of the same brand can be a great way to increase your available VRAM to load larger models.  When you load a new model, Ollama evaluates the required VRAM for the model against what is currently available.  If the model will entirely fit on any single GPU, Ollama will load the model on that GPU.  This typically provides the best performance as it reduces the amount of data transfering across the PCI bus during inference.  If the model does not fit entirely on one GPU, then it will be spread across all the available GPUs.
+
+## How can I set the quantization type for the K/V cache?
+
+The K/V context cache can be quantized to significantly reduce memory usage when Flash Attention is enabled.
+
+To use quantized k/v cache with Ollama you can set the following environment variables:
+
+- `OLLAMA_CACHE_TYPE_K` - The quantization type for the key cache.  Default is `f16`.
+- `OLLAMA_CACHE_TYPE_V` - The quantization type for the value cache.  Default is `f16`.
+
+Note: Currently this is a global option - meaning all models will run with the specified quantization type.
+
+While there are [a number of quantization types available](https://github.com/ggerganov/llama.cpp/pull/7527), Ollama currently supports:
+
+- `f32` - full precision and memory usage.
+- `f16` - high precision and memory usage (default).
+- `q8_0` - 8-bit quantization, uses approximately 1/2 the memory of `f16` with a very small loss in precision.
+- `q4_0` - 4-bit quantization, uses approximately 1/4 the memory of `f16` with a small-medium loss in precision.
+
+If you're using an uncommon quant type, you may benefit from build Ollama with `LLAMA_CUDA_FA_ALL_QUANTS=1` to make llama.cpp build all flash attention quantization types.
```

Comment:
Is this more or less documentation for `OLLAMA_FLASH_ATTENTION`? Good to know regardless

Reply:
Yeah you're probably right it applies to FA and K/V cache quantisation, I'll move it.

Reply:
Done, and documented the basic enabling of Flash Attention 🤣